deploy/charts/rook-ceph-cluster/prometheus/localrules.yaml

```diff
@@ -0,0 +1,890 @@
+groups:
```

Comment:
This yaml is directly copied from the [ceph source repo](https://github.com/ceph/ceph/blob/pacific/monitoring/prometheus/alerts/ceph_default_alerts.yml). This way, we (hopefully) don't have to maintain any customizations to what is picked up from the ceph repo.